tools/releases/version.sh

```diff
@@ -1,31 +1,5 @@
 #!/usr/bin/env bash
 
-function envoy_version() {
-  set -o errexit
-  set -o pipefail
-  set -o nounset
-  # Returns Envoy version by ENVOY_TAG:
-  # - if ENVOY_TAG is a real git tag like 'v1.20.0' then the version is equal to '1.20.0' (without the first letter 'v').
-  # - if ENVOY_TAG is a commit hash then the version will look like '1.20.1-dev-b16d390f'
-
-  ENVOY_TAG=${ENVOY_TAG:-"v1.29.5"}
-  ENVOY_VERSION=$(curl --silent --location "https://raw.githubusercontent.com/envoyproxy/envoy/${ENVOY_TAG}/VERSION.txt")
-
-  # for envoy versions older than v1.22.0 file 'VERSION.txt' used to be called 'VERSION'
-  if [[ "${ENVOY_VERSION}" == "404: Not Found" ]]; then
-    ENVOY_VERSION=$(curl --silent --location --fail "https://raw.githubusercontent.com/envoyproxy/envoy/${ENVOY_TAG}/VERSION")
-  fi
-
-  if [[ "${ENVOY_TAG}" =~ ^v[0-9]*\.[0-9]*\.[0-9]*$ ]]; then
-    echo "${ENVOY_VERSION}"
-  else
-    echo "${ENVOY_VERSION}-${ENVOY_TAG:0:8}"
-  fi
-  set +o errexit
-  set +o pipefail
-  set +o nounset
-}
-
```

Comment:
are we sure we don't need any of this? the only thing that boggles my mind is pulling `https://raw.githubusercontent.com/envoyproxy/envoy/${ENVOY_TAG}/VERSION.txt` - it sometimes has `-dev` (on main) otherwise it always is just the tag 🤔 did we ever pull `-dev` versions?

Reply:
No, we never use dev envoy version